src/webapp/pages/form-page/useForm.ts

```diff
@@ -62,15 +61,12 @@ export function useForm(formType: FormType, id?: Id): State {
     const [globalMessage, setGlobalMessage] = useState<Maybe<GlobalMessage>>();
     const [formState, setFormState] = useState<FormLoadState>({ kind: "loading" });
     const [configurableForm, setConfigurableForm] = useState<ConfigurableForm>();
-    const [currentEventTrackerState, setCurrentEventTrackerState] =
-        useState<Maybe<DiseaseOutbreakEvent>>();
+
     const [formLabels, setFormLabels] = useState<FormLables>();
     const [isLoading, setIsLoading] = useState(false);
     const currentEventTracker = getCurrentEventTracker();
 
     useEffect(() => {
-        if (currentEventTrackerState?.id === currentEventTracker?.id) return;
```

Comment:
This is needed to avoid infinite loop but fix it with: 
```typescript
if (
            currentEventTrackerState &&
            currentEventTracker &&
            currentEventTrackerState?.id === currentEventTracker?.id
        )
            return;
``` 

[Screencast from 2024-10-24 16-38-04.webm](https://github.com/user-attachments/assets/a8f3ce0d-78bb-4c0f-8308-aa96f876c0d5)